RELEASES.md

```diff
@@ -0,0 +1,55 @@
+# Releases
+
+This document describes how Strimzi handles versioning, releases new versions and communicates about them.
+
+Strimzi uses [semantic versioning](https://semver.org/) as a versioning schema for the releases of all the components.
+
+## Release Cycle
+
+Strimzi is using a release schedule of 1 or 2 months for the "core" component, the cluster operator.
+The [GitHub milestones](https://github.com/strimzi/strimzi-kafka-operator/milestones) are used to track the features and bug fixes included in each release.
+There is also a [roadmap](https://github.com/orgs/strimzi/projects/1) to describe the features that will be available in the future releases.
+
+The other components, under the Strimzi umbrella, are released on a case-by-case basis depending on the new features added, bugs and vulnerabilities fixed.
+
+## Release Process
+
+Most of the Strimzi components have a corresponding documentation about how to make a new release.
```

Comment:
```suggestion
Most of the Strimzi components have documentation about how to make a new release.
```

Can we make this `RELEASE.md` for each component? It would make this document seem less _laissez faire_.

Reply:
What I meant was, can we make `RELEASE.md` the standard for all our projects, so that this sentence can read 
```suggestion
All Strimzi components have a documentation for their release process in their `RELEASE.md` file.
```

Reply:
Currently each component has a document like this ... i.e. bridge
https://github.com/strimzi/strimzi-kafka-bridge/blob/main/development-docs/RELEASING.md
It explains the steps for making the release. While you want this RELEASE.md duplicated across all repos with the difference in the process for each one? :-(

Reply:
I think one of the two of us is confused. It might be me. But I think if every project already has a `RELEASING.md` file, then all I'm asking for here is that we say that here. The problem with "Most of the Strimzi components have documentation..." is it raises the question "Why don't all of them". And the current sentence doesn't make it easier for a reader to know where this documentation lives. So I'm just asking that we state what the file name is.

Reply:
Ok I was making the point about the difference between the current `RELEASES.md` and the `RELEASING.md` file we have in the other repository.
The `RELEASING.md` describe the actual steps to do a build (i.e. branch creation, tagging, Azure pipeline runs and so on).
The `RELEASES.md` is about having an overall description of how releases and versioning work in Strimzi.
I thought you were referring to have a `RELEASES.md` for each repository.

Anyway I just double checked. Operator, bridge and canary having a `RELEASING.md`. OAuth calls is `RELEASE.md`. Other repos (i.e. quota plugin) don't have something like that.

Reply:
Let's change the Oauth `RELEASE.md` to `RELEASING.md` for consistency (and because `RELEASE.md` is confusingly close to `RELEASES.md`, which we've just established has a different purpose).

Reply:
Agree. Just opened https://github.com/strimzi/strimzi-kafka-oauth/pull/139